vipps-recurring-api.md

```diff
@@ -234,13 +242,12 @@ in the Getting started guide, for details.
 
 ## Idempotency Key header (V3 API Coming Soon)
 
-The `Idempotency-Key` header must be set in each `POST` or `PATCH` request.
-This way, if a request fails for any reason, it can be retried with the same `Idempotency-Key`.
-Also, in the case of retries, it will prevent duplicating operations.
+The `Idempotency-Key` header must be set in any request that creates or modifies a resource (`POST`, `PUT`, `PATCH` or `DELETE`).
+This way, if a request fails for any technical reason, or there is a networking issue, it can be retried with the same `Idempotency-Key`. The idempotency-key should prevent operations and side-effects from being performed more than once, and you should receive the same response as if you only sent one request.
 
-See the
```

Comment:
I wonder if we should keep this reference and possibly move the *Important* messages to that global definition place ?
The push is to move common stuff out of the different API guides and add links in an effort to shorten the guides.
I have a list of several other things I am going to try and move out. What is your opinion?

Reply:
If you can move the important stuff out to the global definition, that would probably be nice. The reason we removed it was that it did not add information, but did add some confusion since it uses a different name for the header (`Request-Id`).

Reply:
I see that. Maybe I can rewrite that global definition to use 'Idempotency-key' as the main name and just add ('sometimes called 'Request-Id') as a note.  And then add those 2 notes into that global page. Then, you could keep the link... ?

Reply:
Like this?  https://github.com/vippsas/vipps-developers/pull/209

Reply:
Maybe, unfortunately I am not sure whether the two **important** notices are valid for all vipps services. They are just how we are implementing it. Also not sure if it is the case that older apis use Request-Id while new apis use Idempotency-Key 🙈😅

Reply:
Let's just keep them in the recurring guide and don't worry.